core/src/test/scala/com/microsoft/azure/synapse/ml/nbtest/SharedNotebookE2ETestUtilities.scala

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.nbtest
+
+import com.microsoft.azure.synapse.ml.build.BuildInfo
+import com.microsoft.azure.synapse.ml.core.env.FileUtilities
+import org.apache.commons.io.FileUtils
+
+import java.io.File
+import java.lang.ProcessBuilder.Redirect
+import scala.sys.process._
+
+
+object SharedNotebookE2ETestUtilities {
+  val ResourcesDirectory = new File(getClass.getResource("/").toURI)
+  val NotebooksDir = new File(ResourcesDirectory, "generated-notebooks")
+
+  def generateNotebooks(): Unit =
+  {
+    cleanUpGeneratedNotebooksDir()
+
+    FileUtilities.recursiveListFiles(FileUtilities
+    .join(BuildInfo.baseDirectory.getParent, "notebooks/features")
+    .getCanonicalFile)
+    .filter(_.getName.endsWith(".ipynb"))
+    .map { f =>
+      FileUtilities.copyFile(f, NotebooksDir, true)
+      val newFile = new File(NotebooksDir, f.getName)
+      val targetName = new File(NotebooksDir, f.getName.replace(" ", "").replace("-", ""))
+      newFile.renameTo(targetName)
+      targetName
+    }
+
+    runCmd(activateCondaEnv ++ Seq("jupyter", "nbconvert", "--to", "python", "*.ipynb"), NotebooksDir)
+  }
+
+  def cleanUpGeneratedNotebooksDir(): Unit =
+  {
```

Comment:
sorry to be a PITA but this is C# style bracketing to put on new line, can you put these brackets after = here and elsewhere?